crates/ra_assists/src/handlers/fix_visibility.rs

```diff
@@ -179,11 +179,7 @@ fn target_data_for_def(
 
 fn vis_offset(node: &SyntaxNode) -> TextSize {
     node.children_with_tokens()
-        .skip_while(|it| match it.kind() {
-            WHITESPACE | COMMENT | ATTR => true,
-            _ => false,
-        })
-        .next()
+        .find(|it| !matches!(it.kind(), WHITESPACE | COMMENT | ATTR))
         .map(|it| it.text_range().start())
         .unwrap_or_else(|| node.text_range().start())
 }
```

Comment:
Let’s pull both fidenticsl unctions up to until

Reply:
Done
https://github.com/rust-analyzer/rust-analyzer/pull/5105/commits/503f9498cd469c6467250c02bd8f6dbce973dd57